build/templates/Dockerfile.template

```diff
@@ -0,0 +1,30 @@
+FROM library/wordpress:${wordhat.wordpress-version}-php${wordhat.php-version}
+
+# change the port to 8080 so we don't have to run as root locally
+RUN sed -i '1 s/\*\:80/\*\:8080/' /etc/apache2/sites-available/000-default.conf
+RUN sed -i 's/Listen 80/Listen 8080/' /etc/apache2/ports.conf
+EXPOSE 8080
+
+RUN apt-get -y update && apt-get -y --force-yes install gnupg && apt-key update  
+RUN apt-get install -y --force-yes netcat mysql-client zip unzip
+
+RUN curl https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer | php -- && mv composer.phar /usr/local/bin/composer
+#RUN yes | pecl install xdebug \
+#    && echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" > /usr/local/etc/php/conf.d/xdebug.ini \
```

Comment:
Do we need xdebug?

Reply:
its not been release for 7.2 yet, and it breaks phpstan, so I've left it out for the moment.

Reply:
ok nice